react-components/src/architecture/base/utilities/colors/getNextColor.test.ts

```diff
@@ -0,0 +1,45 @@
+import { describe, expect, test } from 'vitest';
+import { getNextColor, getNextColorByIndex } from './getNextColor';
+import { type Color } from 'three';
+
+describe('colorExtensions', () => {
+  describe(getNextColor.name, () => {
+    test('should different', () => {
+      let prevColor = getNextColor();
+      for (let i = 0; i < 20; i++) {
+        const color = getNextColor();
+        expect(prevColor.equals(color)).toBe(false);
```

Comment:
You can probably use `expect(..).not.toBeEqual(...)` here? Or is the `equals`-function you're using doing something else?